client/galaxy/scripts/apps/analysis.js

```diff
@@ -83,6 +84,7 @@ window.app = function app( options, bootstrapped ){
             '(/)user(/)(:form_id)' : 'show_user_form',
             '(/)workflow(/)' : 'show_workflows',
             '(/)pages(/)(:action_id)' : 'show_pages',
+            '(/)histories(/)(:action_id)' : 'show_histories',
```

Comment:
Similar to the comment here https://github.com/galaxyproject/galaxy/pull/4163#discussion_r121369351, this routes /history/*/ to /history/list/.  This isn't good practice, invalid URLs should return error codes, instead of routing to something that wasn't requested.